src/main/java/picard/arrays/GtcToVcf.java

```diff
@@ -358,41 +358,23 @@ private VariantContext makeVariantContext(Build37ExtendedIlluminaManifestRecord
-        builder.attribute(InfiniumVcfFields.DEV_THETA[InfiniumVcfFields.GENOTYPE_VALUES.BB.ordinal()], formatFloatForVcf(egtFile.devThetaBB[egtIndex]));
-        builder.attribute(InfiniumVcfFields.MEAN_THETA[InfiniumVcfFields.GENOTYPE_VALUES.AA.ordinal()], formatFloatForVcf(egtFile.meanThetaAA[egtIndex]));
-        builder.attribute(InfiniumVcfFields.MEAN_THETA[InfiniumVcfFields.GENOTYPE_VALUES.AB.ordinal()], formatFloatForVcf(egtFile.meanThetaAB[egtIndex]));
-        builder.attribute(InfiniumVcfFields.MEAN_THETA[InfiniumVcfFields.GENOTYPE_VALUES.BB.ordinal()], formatFloatForVcf(egtFile.meanThetaBB[egtIndex]));
-
-        EuclideanValues aaVals = polarToEuclidean(egtFile.meanRAA[egtIndex], egtFile.devRAA[egtIndex],
-                egtFile.meanThetaAA[egtIndex], egtFile.devThetaAA[egtIndex]);
-        EuclideanValues abVals = polarToEuclidean(egtFile.meanRAB[egtIndex], egtFile.devRAB[egtIndex],
-                egtFile.meanThetaAB[egtIndex], egtFile.devThetaAB[egtIndex]);
-        EuclideanValues bbVals = polarToEuclidean(egtFile.meanRBB[egtIndex], egtFile.devRBB[egtIndex],
-                egtFile.meanThetaBB[egtIndex], egtFile.devThetaBB[egtIndex]);
-
-        builder.attribute(InfiniumVcfFields.DEV_X[InfiniumVcfFields.GENOTYPE_VALUES.AA.ordinal()], formatFloatForVcf(aaVals.devX));
-        builder.attribute(InfiniumVcfFields.DEV_X[InfiniumVcfFields.GENOTYPE_VALUES.AB.ordinal()], formatFloatForVcf(abVals.devX));
-        builder.attribute(InfiniumVcfFields.DEV_X[InfiniumVcfFields.GENOTYPE_VALUES.BB.ordinal()], formatFloatForVcf(bbVals.devX));
-        builder.attribute(InfiniumVcfFields.MEAN_X[InfiniumVcfFields.GENOTYPE_VALUES.AA.ordinal()], formatFloatForVcf(aaVals.meanX));
-        builder.attribute(InfiniumVcfFields.MEAN_X[InfiniumVcfFields.GENOTYPE_VALUES.AB.ordinal()], formatFloatForVcf(abVals.meanX));
-        builder.attribute(InfiniumVcfFields.MEAN_X[InfiniumVcfFields.GENOTYPE_VALUES.BB.ordinal()], formatFloatForVcf(bbVals.meanX));
-        builder.attribute(InfiniumVcfFields.DEV_Y[InfiniumVcfFields.GENOTYPE_VALUES.AA.ordinal()], formatFloatForVcf(aaVals.devY));
-        builder.attribute(InfiniumVcfFields.DEV_Y[InfiniumVcfFields.GENOTYPE_VALUES.AB.ordinal()], formatFloatForVcf(abVals.devY));
-        builder.attribute(InfiniumVcfFields.DEV_Y[InfiniumVcfFields.GENOTYPE_VALUES.BB.ordinal()], formatFloatForVcf(bbVals.devY));
-        builder.attribute(InfiniumVcfFields.MEAN_Y[InfiniumVcfFields.GENOTYPE_VALUES.AA.ordinal()], formatFloatForVcf(aaVals.meanY));
-        builder.attribute(InfiniumVcfFields.MEAN_Y[InfiniumVcfFields.GENOTYPE_VALUES.AB.ordinal()], formatFloatForVcf(abVals.meanY));
-        builder.attribute(InfiniumVcfFields.MEAN_Y[InfiniumVcfFields.GENOTYPE_VALUES.BB.ordinal()], formatFloatForVcf(bbVals.meanY));
+
+        for (InfiniumVcfFields.GENOTYPE_VALUES gtValue : InfiniumVcfFields.GENOTYPE_VALUES.values()) {
```

Comment:
extract `gtValue.ordinal()` to a constant at the top of the loop

Reply:
Done.